### DISCUSSION TITLE
Meta-mot

### DISCUSSION THREAD

**Discussion by Jireck-npds** [npds/npds_dune#542] [General]
Hello 
Je suis etonné de l'existance de ces meta-mots :  
<html>
<body>

  |!bgcolor1! | meta | variable global $bgcolor1 (ancien theme)
  | !bgcolor2! | meta | variable global $bgcolor2 (ancien theme)
  | !bgcolor3! | meta | variable global $bgcolor3 (ancien theme)
  | !bgcolor4! | meta | variable global $bgcolor4 (ancien theme)
  | !bgcolor5! | meta | variable global $bgcolor5 (ancien theme)
  | !bgcolor6! | meta | variable global $bgcolor6 (ancien theme)
  |!closetable! | meta | Appel la fonction CloseTable de NPDS
  |!textcolor1! | meta | variable global $textcolor1 (ancien theme)
  | !textcolor2! | meta | variable global $textcolor2 (ancien theme)

</body>
</html>

**Comment by jipexu** [comment]
? sauf erreur ces variables ont été supprimées ?

**Comment by Jireck-npds** [comment]
oui
